<!doctype html>

<!--[if lt IE 7]>

<html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="en">

<![endif]-->
<!--[if IE 7]>

<html class="no-js lt-ie9 lt-ie8" lang="en">

<![endif]-->
<!--[if IE 8]>

<html class="no-js lt-ie9" lang="en">

<![endif]-->
<!--[if gt IE 8]>
<!-->
<html class="no-js" lang="en">
    <!--
  <![endif]-->
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <title>
            Bioviz - News
        </title>
        <meta name="description" content="">
        <meta name="author" content="">

        <meta name="viewport" content="width=device-width">
        <!--For use in production only... compile to css before release -->
        <link href="css/bootstrap.css" rel="stylesheet">
        <link href="css/bootstrap-glyphicons.css?v=1.3" rel="stylesheet">
        <link href="bower_components/font-awesome/css/font-awesome.css" rel="stylesheet">
        <link rel="stylesheet" href="less/style.css?v=1.3">
        <link rel="stylesheet" href="bower_components/lightbox2/dist/css/lightbox.css">
        <script src="js/libs/modernizr-2.5.3-respond-1.1.0.min.js?v=1.3">
        </script>
    </head>
    <body>
        <div id="TopMenu">
        </div>

        <div class="container main-content">            
            <div id="markupOutput"> </div>
            <h3>
                December 15, 2014: IGB 8.2 released
            </h3>
            <p>
                The IGB development team is pleased to announce the immediate availability of the new IGB 8.2 major release. This release contains resolutions to over 200 issues, 
                including many bug fixes and new features. 
            </p>
            <h4>Major New Features:</h4>
            <ul>
                <li>Paired End Visualization for NGS data</li>
                <li>Addition of the latest fruit fly genome</li>
            </ul>
            <h4>Usability Improvement Highlights:</h4>
            <ul>
                <li>New tooltips</li>
                <li>New Right Click Context Menu available in track "whitespace"</li>
                <li>Right-click menu re-organization (easier to find stuff)</li>
                <li>Improved custom genome loading widget</li>
                <li>Improved Image Export widgets</li>          
            </ul>
            <h4>Power User Highlights:</h4>
            <ul>
                <li>New feature available for controlling IGB from R (Sample code available at <a href="https://bitbucket.org/lorainelab/rtoigb">https://bitbucket.org/lorainelab/rtoigb</a>)</li>       
            </ul>
            <h4>Engineering Notes:</h4>
            <ul>
                <li>Migration of build system from ant to maven with many significant improvements for developers
                    <ul>
                        <li>Maven is more natural fit for our modular osgi project and having a plugin archetype reduces the technical burden to community contributors</li>
                        <li>OSGI manifest files no longer need to be manually managed since the maven bundle plugin leverages a projects declared dependencies and classpath to populate the manifest<br>
                            project dependencies are formalized and versions are now tracked (a major improvement over a set of jars in a lib directory)</li>
                        <li>hosted maven repository contains project artifacts</li>
                    </ul>
                </li>
                <li>Upgraded internal logging system to use standard slf4j api (with logback implementation)
                    <ul>
                        <li>SLF4j is a powerful api which greatly improves on java.util.logging and will improve our code quality and help reduce the time it takes to debug complex issues</li>
                    </ul>
                </li>
                <li>Fixed External Contributor Plugin Repository System
                    <ul>
                        <li>This is a repository which makes available plugins which are not turned on by default in IGB.  These plugins are not on by default either because they are not yet stable enough to be a part of the core project or because they add functionality not needed by the majority of our users.  Restoring our projects ability to support community plugins was an important development priority for us while preparing IGB 8.2.</li>
                    </ul>
                </li>
            </ul>
            <hr>
            <h3>
                December 8, 2014: IGB 8.1.13 released
            </h3>
            <p>
                In preparation for our upcoming IGB 8.2 release, we have released a minor release with some minor bug fixes to our update notification system.  With this notification system we will be able to notify users when we release 
                IGB 8.2.  This is important, because major releases cannot be pushed out silently, and require users to download a fresh copy of our software.
            </p>
            <ul>
                <li>Patch bug in Notification System so it can be leveraged to notify user's of IGB 8.2 release (<a href="https://jira.transvar.org/browse/IGBF-282">IGBF-282</a>)</li>
            </ul>
            <hr>
            <h3>
                December 1, 2014: Updated JNLP Generator Service Released
            </h3>
            <p>
                In preparation for our upcoming IGB 8.2 release, we have released an updated and simplified jnlp generator web service. For more information, see: <a href="jnlpService.html">JNLP Generator Service</a>
            </p>
            <hr>
            <h3>
                November 25, 2014: IGB 8.1.12 Released
            </h3>
            <p>
                We have released a minor patch to ensure our blast functionality is not impacted by an upcoming NCBI blast URL change,  For more information, see:
            </p>
            <ul>
                <li>NCBI BLAST url is changing on Dec. 1, 2014 (<a href="https://jira.transvar.org/browse/IGBF-267">IGBF-267</a>)</li>
            </ul>
            <hr>
            <h3>
                September 10, 2014: IGB 8.1.11 Released
            </h3>
            <p>
                The IGB team is pleased to announce a new release of IGB. This new version fixes an error in SVG image export. Thank you to everyone who reported it. This new release also fixes BLAST search on gene models. Last but not least, we've modified our backup QuickLoad server code. Now, if the main IGBQuickLoad.org site is unavailable, IGB will automatically switch over to our mirror site. For more information on these changes and why we needed them, see:
            </p>
            <ul>
                <li>Mirror site failover (<a href="https://jira.transvar.org/browse/IGBF-141">IGBF-141</a>)</li>
                <li>BLAST search fix (<a href="https://jira.transvar.org/browse/IGBF-186">IGBF-186</a>)</li>
                <li>SVG export functionality (<a href="https://jira.transvar.org/browse/IGBF-185">IGBF-185</a>)</li>
            </ul>
            <hr>
            <h3>Sept 2, 2014: Welcome aboard Nowlan Freese</h3>
            <p>Today we welcome Nowlan Freese, Ph.D., who joins the IGB team as a postdoctoral trainee. Nowlan will help us reach out to new users and also help existing users get greater value from IGB. Nowlan studied developmental biology as a graduate student at Clemson University and most recently did postdoctoral work at the Max Planck Institute. During his postdoc, Nowlan plans to build expertise in bioinformatics data analysis, using IGB and other tools.</p>
            <h3>
                July 30, 2014: IGB 8.1.10 Released
            </h3>
            <p>
                Today we released IGB 8.1.10, a minor release which includes a patch for the following bug
            </p>
            <ul>
                <li>IGB will no longer launch using Java Web Start on Windows 7/8 platform after release of latest Java version (<a href="https://jira.transvar.org/browse/IGBF-136">IGBF-136</a>)</li>
            </ul>
            <hr>
            <h3>
                July 30, 2014: Notice: Windows 7/8 Java Webstart Bug introduced with Java 7 update 65 and Java 8 update 11
            </h3>
            <p>
                We have been able to confirm the latest releases of Java 7 and Java 8 have introduced a windows specific Java Web Start bug which is unfortunately currently impacting our users.  
                This new bug impacts all past web start releases of IGB.  We have submitted a bug report to Oracle, but will be releasing a workaround later today to resolve this issue for our users.
            </p>
            <p>For more details on the bug see our Jira Issue (<a href="https://jira.transvar.org/browse/IGBF-136">IGBF-136</a>)</p>
            <hr>
            <h3>
                July 21, 2014: IGB 8.1.9 Released
            </h3>
            <p>
                Today we released IGB 8.1.9, a minor release which includes a patch for the following
            </p>
            <ul>
                <li>Typo in View menu entry for "Show Zoom Stripe" (<a href="https://jira.transvar.org/browse/IGBF-124">IGBF-124</a>)</li>
            </ul>
            <hr>
            <h3>
                July 08, 2014: IGB 8.1.8 Released
            </h3>
            <p>
                Today we released IGB 8.1.8, a minor release which includes a patch for the following bug
            </p>
            <ul>
                <li>Adding local QuickLoad site fails (<a href="https://jira.transvar.org/browse/IGBF-95">IGBF-95</a>)</li>
            </ul>
            <hr>
            <h3>
                June 16, 2014: IGB 8.1.7 Released
            </h3>
            <p>
                Today we released IGB 8.1.7, a minor release which includes a patch for the following bug
            </p>
            <ul>
                <li>Unable to load gzipped wig files over https (<a href="https://jira.transvar.org/browse/IGBE-171">IGBE-171</a>)</li>
            </ul>
            <hr>      
            <h3>
                June 10, 2014: IGB 8.1.5 Released
            </h3>
            <p>
                Today we released IGB 8.1.5, a minor release which addresses many small bugs and improves stability. The list below highlights some of the improvements.
            </p>
            <ul>
                <li>Starting IGB without an Internet connection no longer causes servers to be permanently disabled</li>
                <li>Quickload mirror site failover has been restored</li>
                <li>IGB Bookmarks embedded in a web page again return an http 204 response code</li>
                <li>The image export widget has been enhanced to improve integration with Mac OS X</li>
            </ul>

            <hr>
            <h3>
                May 30, 2014: IGB 8.1.4 Released
            </h3>
            <p>
                Today we released IGB 8.1.4, a minor release which a bug impacting the proper loading of BAM files from https locations
            </p>
            <hr>
            <h3>
                May 29, 2014: IGB 8.1.3 Released
            </h3>
            <p>
                Today we released IGB 8.1.3, a minor release which resolves several bugs and enhances the look and feel of several widgets for MAC OSX users.
            </p>
            <ul>
                <li>Fix for track name bug when bed files had the same trackline name information (<a href="https://jira.transvar.org/browse/IGBE-160">IGBE-160</a>)</li>
                <li>Fix for annotation name black rectangle bug (<a href="https://jira.transvar.org/browse/IGBF-31">IGBF-31</a>)</li>
                <li>Fix for read sequence rendering bug (<a href="https://jira.transvar.org/browse/IGBF-31">IGBF-31</a>)</li>
                <li>Workaround for  Safari caching Galaxy External View links bug added<a href="https://jira.transvar.org/browse/IGBF-44">IGBF-44</a></li>
            </ul>
            <hr>
            <h3>
                April 16, 2014: IGB 8.1.2 Released
            </h3>
            <p>
                Today we released IGB 8.1.2, a minor release which resolves a critical bug
            </p>
            <hr>
            <h3>
                April 15, 2014: IGB 8.1 Released
            </h3>
            <p>
                Today we released IGB 8.1, a major release which includes enhanced integration with <a href="http://galaxyproject.org/">Galaxy</a>.
            </p>
            <hr>
            <h3>
                March 19, 2014: IGB 8.0.4 Released
            </h3>

            <p>
                Today we released IGB 8.0.4, a minor release which retracts some functionality which was not meant to be included in the 8.0.3 release.
            </p>
            <hr>
            <h3>
                March 18, 2014: IGB 8.0.3 Released
            </h3>

            <p>
                Today we released IGB 8.0.3, a minor release which includes a few bug fixes as well as added support for custom jnlp launch files.  
                We have also added a new 
                <a href="jnlpService.html">JNLP Generation Service</a> to bioviz.org to help facilitate the generation of custom jnlp launch files.
            </p>
            <hr>

            <h3>
                March 7, 2014: Dec 2013 human genome release (hg38) now supported
                in IGB
            </h3>

            <p>
                Today we add the latest human genome assembly (hg38) as
                H_sapiens_Dec_2013 to the IGB QuickLoad data repository. Now, when you
                visit the latest human genome by clicking the Mona Lisa image on the
                IGB start screen, the Dec. 2013 human genome assembly and RefGene
                annotations will automatically load. You can still view older versions
                by selecting them in the 
                <b>
                    Current Genome
                </b>
                tab. If you have
                questions, please feel free to get in touch! 
                <a href=
                   "https://twitter.com/igbbioviz">
                    Send us a tweet</a>
                or post a message
                on the 
                <a href="http://sourceforge.net/p/genoviz/discussion/439787">
                    IGB
                    Discussion forum at SourceForge</a>.
            </p>
            <hr>

            <h3>
                March 3, 2014: IGB moved to git repository hosted at
                BitBucket.org
            </h3>

            <p>
                Today, after many years of using svn, we migrated the IGB source
                code to a new 
                <a href=
                   "https://bitbucket.org/lorainelab/integrated-genome-browser">
                    git
                    repository</a>
                hosted on BitBucket. Using a distributed version control
                system git will make it easier for developers to contribute new ideas
                and new features to the project. Thank you David for managing the
                transition.
            </p>
            <hr>

            <h3>
                March 3, 2014: IGB 8.0.2 released
            </h3>

            <p>
                Today we released IGB 8.0.2, a minor release that corrects issues
                affecting heatmap graph style display, search, and bookmark
                exporting.
            </p>
            <hr>

            <h3>
                Feb. 24, 2014: IGB team welcomes new lead developer David
                Norris
            </h3>

            <p>
                UNC Charlotte Computer Science graduate David Norris re-joined the
                team today, taking over from Hiral Vora as lead developer on the IGB
                project. In his previous position, David worked at Red Hat on multiple
                projects and gained valuable experience in many aspects of open source
                software development. In his new position, he looks forward to using
                his knowledge and skill to improve IGB functionality for users. Welcome
                to the team!
            </p>
            <hr>

            <h3>
                Feb. 8, 2014: SR gene annotations updated for Arabidopsis TAIR10
                genome
            </h3>

            <p>
                SR genes encode RNA-binding proteins that regulated and help to
                catalyze splicing. We've updated the gene symbols and descriptions for
                Arabidopsis SR genes so that IGB now follows the nomenclature
                recommendations from 
                <a href=
                   "http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2965536/">
                    Barta et al
                    (2010) Plant Cell</a>. Code used to modify the annotation files is
                freely available from 
                <a href=
                   "https://bitbucket.org/lorainelab/sr-gene-symbol-update">
                    this
                    repository</a> at 
                <a href=
                   "http://www.bitbucket.org">
                    BitBucket.org</a>.
            </p>
            <hr>

            <h3>
                Feb. 6, 2014: Older human genomes supported in
                IGBQuickload.org
            </h3>

            <p>
                In response to user requests, we've added two archival human genome
                assemblies (hg17 and hg18) to the IGBQuickload site. This will enable
                you to view gene model (refGene) annotations with HUGO gene symbols for
                these older releases similar to the latest release.
            </p>
            <hr>

            <h3>
                Jan. 3, 2014: Farewell Hiral
            </h3>

            <p>
                Today was Hiral's last day on the IGB team. After four years of
                great work, he is leaving for a new position at Deutsches Bank in the
                Research Triangle Park area. Thank you for your many outstanding
                efforts helping scientists get the most value from their data. Good
                luck in your new position!
            </p>
            <hr>

            <h3>
                Dec. 18, 2013: DAS2 retired
            </h3>

            <p>
                Today we retired the DAS2 servers hosted on IGBQuickload.org and
                BioViz.org. We also retired the Affymetrix Arabidopsis tiling array
                data sets as well as Affymetrix microarray probe set visualizations for
                most genome versions. Note that IGB can still open Affymetrix
                "link.psl" files, and so if you need to visualize the probe sets, you
                can download the files from Affymetrix and open them in IGB.
            </p>
            <hr>

            <h3>
                Dec. 17, 2013: Farewall Alyssa!
            </h3>

            <p>
                Today we say a sad "goodbye" to IGB Outreach and Testing Coordinator
                Alyssa Gulledge, who leaves us for a new opportunities in Atlanta.
                Thank you Alyssa for your many efforts and we wish you the best of
                luck. We will miss you!
            </p>
            <hr>

            <h3>
                Dec. 6, 2013: IGB 8 released
            </h3>

            <p>
                This new version of IGB has many improvements, including:
            </p>

            <ul>
                <li>
                    <b>
                        Faster data loading.
                    </b>
                    New optimizations enable gene
                    models to load even faster from IGBQuickLoad servers.
                </li>

                <li>
                    <b>
                        Track filters.
                    </b>
                    New track filters let you hide or show
                    reads and other features based on attributes like splicing and
                    score.
                </li>

                <li>
                    <b>
                        Color by score improved.
                    </b>
                    IGB's new Cytoscape-inspired
                    gradient color chooser lets you choose a range of colors to
                    indicate scores. This is especially useful for analyzing ChIP-Seq
                    results and output from programs like MACS.
                </li>

                <li>
                    <b>
                        Right-click blast searches.
                    </b>
                    Now you can right-click a
                    gene model or other features and search its protein or spliced
                    nucleotide sequence against the NCBI nr protein database (inspired
                    by Apollo)
                </li>
            </ul>
            <hr>

            <h3>
                Oct. 30 - Nov. 2, 2013: IGB at Genome Informatics, CSHL
            </h3>

            <p>
                Ann 
                <a href="GI-CHSL-2013.pdf">
                    presented a poster</a>
                and gave demos
                introducing IGB 8 to attendees at the Cold Spring Harbor Genome
                Informatics Meeting. To test-run IGB 8, download and run 
                <a href=
                   "http://bioviz.org/igb/igb_preview.shtml">
                    IGB 8 preview</a>.
            </p>
            <hr>

            <h3>
                Oct. 29, 2013: Focus on a Feature - Introducing IGB 8
            </h3>

            <p>
                This month hosted 
                <i>
                    two sessions
                </i>
                of IGB Focus on a Feature to
                introduce new features of IGB 8, scheduled for release in December. To
                test-run IGB 8, download and run 
                <a href=
                   "http://bioviz.org/igb/igb_preview.shtml">
                    IGB 8 preview</a>.
            </p>

            <p>
                In this live tutorial, we showed how to:
            </p>

            <ul>
                <li>
                    Load your own custom genome into IGB
                </li>

                <li>
                    Load and filter RNA-Seq data
                </li>

                <li>
                    Color-code features by score
                </li>

                <li>
                    Run BLASTX searches at NCBI
                </li>

                <li>
                    And more...
                </li>
            </ul>

            <p>
                Also see the 
                <a href=
                   "http://www.biostars.org/p/84681/">
                    Announcement</a> on the BioStars message board.
            </p>
            <hr>

            <h3>
                Sept. 25, 2013: Focus on a Feature - Working with Sequence
                Data
            </h3>
            <br>

            <p>
                Alyssa Gulledge and Ann Loraine led an hour-long on-line 
                <b>
                    Focus on
                    a Feature
                </b>
                webinar that introduced participants to some neat tricks
                you can do with the 
                <b>
                    IGB Advanced Search
                </b>
                tab, such as searching
                for transcription factor binding sites and mapping and visualizing PCR
                primer sequences. Ann started the webinar with a quick review of moving
                and zooming in IGB, and then Alyssa showed off the sequence
                manipulation and analysis features. Due to technical problems, we were
                only able to record Alyssa's part of the presentation. We'll post a
                video on the 
                <a href=
                   "http://www.youtube.com/channel/UC0DA2d3YdbQ55ljkRKHRBkg">
                    IGB YouTube
                    channel</a> as soon as we finish editing it. For a description of the
                webinar, see our 
                <a href=
                   "http://www.biostars.org/p/81896">
                    announcement</a>
                on the 
                <a href=
                   "http://www.biostars.org">
                    Biostars Web site</a>.

            </p>
            <hr>

            <h3>
                Sept. 9, 2013: IGB 7.0.4 released
            </h3>
            <br>

            <p>
                Today a new bug fix release debuted on the Bioviz Web site. No new
                features were introduced, only corrections to existing features and
                optimizations to improve IGB performance and stability.
            </p>
            <hr>

            <h3>
                Aug. 28, 2013: First Focus on a Feature - Graphs
            </h3>
            <br>

            <p>
                To help IGB users get more out IGB, we've started a new monthly
                <b>
                    Focus on a Feature
                </b>
                webinar to be held at 1:30 pm Eastern time on
                the last Wednesday of every month. The first 
                <b>
                    Focus on a Feature
                </b>
                event covered genome graphs and graph manipulations.
            </p>
            <hr>

            <h3>
                July 22, 2013: IGB 7.0.3 released
            </h3>
            <br>

            <p>
                A new bug fix release was released today. As always, this release
                contains no new features, just fixes for bugs reported by users or the
                IGB testing team.
            </p>
            <hr>

            <h3>
                May 17, 2013: IGB Workshop at WiNGS
            </h3>
            <br>

            <p>
                The IGB team led a hands-on workshop demonstrating how to use IGB at
                the 
                <a href="http://cci.uncc.edu/wings">
                    Workshop in Next-Generation
                    Sequencing and Metabolomics (WiNGS)</a> held at the UNC Charlotte
                Center City campus on May 17. This two-hour workshop described using
                IGB to explore RNA-Seq data set from pollen, based on our article
                titled 
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/23590974">
                    RNA-seq of
                    Arabidopsis pollen uncovers novel transcription and alternative
                    splicing</a>. A step-by-step 
                <a href=
                   "https://pods.iplantcollaborative.org/wiki/display/WiNGS2013/Integrated+Genome+Browser">
                    Web-based tutorial from the workshop</a> is available.
            </p>
            <hr>

            <h3>
                April 30, 2013: IGB 7.0.2 released
            </h3>
            <br>

            <p>
                Today we released a new "bug fix" release of IGB. This releases
                introduces no new features, just fixes for bugs reported by users or
                the IGB testing team.
            </p>
            <hr>

            <h3>
                April, 2013: New IGB QuickLoad site for pollen
            </h3>
            <br>

            <p>
                A new IGB QuickLoad site for pollen-related data sets is now
                available at 
                <a href=
                   "http://igbquickload.org/pollen">
                    igbquickload.org/pollen</a>. The
                inaugural data set on the site is a 75-base, single-end RNA-Seq dataset
                from Arabidopsis pollen and seedlings. An in-depth analysis of these
                data appears in 
                <a href=
                   "http://www.ncbi.nlm.nih.gov/pubmed/23590974">
                    RNA-Seq of Arabidopsis
                    pollen uncovers novel transcription and alternative splicing</a>,
                published on-line in the journal 
                <a href=
                   "http://www.plantphysiol.org">
                    Plant Physiology.</a>
            </p>
            <hr>

            <h3>
                Feb. 11, 2013: IGB 7.0.1 released
            </h3>
            <br>

            <p>
                A new minor release was released. This releases introduces no new
                features, just fixes for bugs reported by users or the IGB testing
                team. For the release announcement, visit 
                <a href="http://sourceforge.net/p/genoviz/mailman/message/30473329/">
                    this post</a> on the GenoViz developers' list.
            </p>
            <hr>

            <h3>
                January 28, 2013: IGB 7.0 released
            </h3>
            <br>

            <p>
                A new major release of IGB is available for download. Some of the
                new features in IGB 7.0 include:
            </p>

            <ul>
                <li>
                    <b>
                        IGB sessions
                    </b>
                    : You can now save and load IGB sessions,
                    which helps you save your place in a long-term analysis or share
                    your work with others.
                </li>

                <li>
                    <b>
                        Sequence viewer
                    </b>
                    : The Sequence Viewer now supports saving
                    and copying protein sequences.
                </li>

                <li>
                    <b>
                        Autoscroll
                    </b>
                    : IGB autoscroll is now easier to configure,
                    start, and stop, which helps in scanning large regions.
                </li>

                <li>
                    <b>
                        Better-looking, more informative gene models
                    </b>
                    : Gene
                    models now have optional intron arrow marks to show strand, and
                    when you load the genomic sequence and zoom in, protein
                    translations appear.
                </li>

                <li>
                    <b>
                        Annotation tab
                    </b>
                    : A new Annotation tab makes configuring
                    and styling annotation tracks much easier. Use the new operations
                    menus to copy, summarize, and compare tracks.
                </li>

                <li>
                    <b>
                        Toolbar
                    </b>
                    : A new configurable toolbar provides one-click
                    access to the most useful IGB functions.
                </li>

                <li>
                    <b>
                        Stack and track height
                    </b>
                    : New options for setting the
                    height of tracks give you more control over the amount and size of
                    data to be shown.
                </li>

                <li>
                    <b>
                        Better searching
                    </b>
                    : Easier, faster searching helps you
                    find genes by gene name or keyword.
                </li>

                <li>
                    <b>
                        Better color by score
                    </b>
                    : You can now set the range of
                    values to show under the color by score annotation setting.
                </li>

                <li>
                    <b>
                        Selection Info box
                    </b>
                    : A new selection info box reports the
                    name or number of annotations you've selected, making it easier to
                    explore your data.
                </li>

                <li>
                    <b>
                        Countless internal improvements
                    </b>
                    : The IGB developers
                    added many improvements to the IGB code base, making IGB 7.0 the
                    most robust IGB ever.
                </li>
            </ul>

            <p>
                For more information, visit 
                <a href=
                   "http://wiki.transvar.org/display/igbman/NEW+in+IGB+7.0">
                    New
                    in IGB 7.0</a>.
            </p>
            <hr>

            <h3>
                July 5, 2012: Fragaria vesca available in IGB QuickLoad
            </h3>
            <br>

            <p>
                Fragaria vesca, also known as the 
                <a href=
                   "http://en.wikipedia.org/wiki/Fragaria_vesca">
                    woodland strawberry</a>,
                is now supported at 
                <a href=
                   "http://www.igbquickload.org">
                    IGBQuickLoad.org</a> as genome version
                F_vesca_Sep_2011. Currently, genomic sequence and gene predictions are
                available. We were able to add the data thanks to advice from Stephen
                Ficklin of the 
                <a href="http://www.bioinfo.wsu.edu/">
                    Main Lab</a> at
                Washington State University. Note that the Main lab develops genome
                database resources for the rosaceae, also known as the rosids,
                including peach, apple, cherry as well as strawberry. For more
                information, visit 
                <a href="http://www.rosaceae.org">
                    GDR - Genome
                    Database for Rosaceae</a>.
            </p>
            <hr>

            <h3>
                June 5, 2012: IGB 6.7.2 released
            </h3>
            <br>

            <p>
                A new minor release of IGB is available. This new release fixes a
                couple of reported bugs and adds new features, including:
            </p>

            <ul>
                <li>
                    support for SVG image format output
                </li>

                <li>
                    an option to specify IGB bookmark server port (see 
                    <a href=
                       "http://sourceforge.net/projects/genoviz/forums/forum/439786/topic/5300474">
                        this IGB forum post</a>)
                </li>
            </ul>
            <hr>

            <h3>
                June 5, 2012: Nematostella vectensis genome available in IGB
                QuickLoad
            </h3>
            <br>

            <p>
                The IGB QuickLoad site now supports the Nematostella vectensis
                genome and annotations. We collected sequence data and annotations from
                <a href="http://www.metazome.net">
                    Metazome</a>, a database similar to
                Phytozome that provides access to a variety of animal genomes. Data
                provided include sequence along with gene model annotations, with
                linkouts to Google and Metazome.
            </p>
            <hr>

            <h3>
                May 17-18, 2012: IGB workshop at WiNGS
            </h3>
            <br>

            <p>
                The IGB team at UNC Charlotte presented at the UNC Charlotte
                Bioinformatics and Genomics Department 
                <a href=
                   "http://cci.uncc.edu/wings">
                    Workshop in Next-Generation Sequencing
                    (WiNGS)</a>. On Thursday, Ann gave a talk describing IGB features and
                demonstrated using IGB to examine splice variants with RNA-Seq data. On
                Friday, she presented a two-hour interactive workshop on using IGB to
                view RNA-Seq data from Arabidopsis. Alyssa Gulledge helped organize the
                WiNGS workshop; she arranged practise talks for speakers, trained
                students, and helped keep things running smoothly. IGB slides and
                workshop materials are available for download:
            </p>

            <ul>
                <li>
                    <a href="WiNGS2012.pdf">
                        Slides from IGB presentation
                    </a>
                </li>

                <li>
                    Workshop tutorial: 
                    <a href="intro_igb.pdf">
                        Introducing
                        IGB
                    </a>
                </li>

                <li>
                    Workshop tutorial: 
                    <a href="novel_splicing.pdf">
                        Novel
                        Splicing
                    </a>
                </li>
            </ul>

            <p>
                If you would like to use the tutorials or slides in teaching, please
                let us know. We would be happy to share the originals with you so that
                you can customize the tutorials for your students.
            </p>
            <hr>

            <h3>
                April 13, 2012: Sorghum RNASeq data set added to
                IGBQuickload
            </h3>
            <br>

            <p>
                We added a new RNAseq data set to the main
                IGBQuickLoad.org/quickload data site. The data are from an article
                titled 
                <a href=
                   "http://www.biomedcentral.com/1471-2164/12/514">
                    Functional annotation
                    of the transcriptome of Sorghum bicolor in response to osmotic stress
                    and abscisic acid</a> published in Oct., 2011.
            </p>

            <p>
                These data include three replicates per sample type of sorghum root
                and shoot samples undergoing simulated drought stress and ABA
                treatments and their corresponding controls. This data set offers a
                unique look at the stress-regulated transcriptome of this highly
                drought-tolerant and hardy plant.
            </p>

            <p>
                To deploy the data set, we downloaded the raw data files as .sra
                files, converted them to FASTQ format, and then aligned them onto the
                January 2009 release of the Sorghum bicolor genome using TopHat 1.3.3.
                We subdivided the alignments files into read alignments that mapped
                once (SM) or more than once (MM) onto the genome. We also generated
                tabix-indexed coverage (.bedgraph) and TopHat junction (.bed.gz) files
                for each sample.
            </p>

            <p>
                To access the data in IGB, load the Sorghum bicolor genome Jan 2009.
                Open the folder named RNASeq. Data sets appear in subfolders SM
                (single- mapping), MM (multi-mapping), ALL (both multi- and
                single-mapping), and Juncs, junctions predicted by TopHat from spliced
                alignments.
            </p>
            <hr>

            <h3>
                Feb 9, 2012: IGB 6.7 released
            </h3>
            <br>

            <p>
                Today we released a new version of IGB, which contains many new
                features and improvements. Just of few of these include:
            </p>

            <ul>
                <li>
                    <b>
                        Improved Data Access Panel
                    </b>
                    : We moved species and genome
                    version selection menus to the Current Genome tab, opening up more
                    space for data set display. We also added new ways to configure
                    tracks via the Data Management Table (formerly called the Load Mode
                    Table).
                </li>

                <li>
                    <b>
                        Take notes with bookmarks
                    </b>
                    : We added the ability to
                    associate notes with bookmarks. We also redesigned the Bookmarks
                    tab and made it easier to sort, delete, and organize your
                    bookmarks.
                </li>

                <li>
                    <b>
                        Better image export functions
                    </b>
                    : You can now specify image
                    resolution and size when you save images from IGB. The Image Export
                    window now also shows you an image preview. We also added the
                    ability to export images from the Genomic Sequence Viewer as well
                    as the Main View and the Sliced View.
                </li>

                <li>
                    <b>
                        Better ways to display results from sequence searches
                    </b>
                    :
                    You can now overlay the results from multiple searches onto the
                    genomic sequence, making it much easier to view PCR primer
                    sequences, promoter motifs, and other items of interest in the same
                    view.
                </li>

                <li>
                    <b>
                        New start screen
                    </b>
                    : IGB now displays image shortcuts you
                    can click to quickly go to your genome of interest.
                </li>

                <li>
                    <b>
                        Better data loading
                    </b>
                    : When you access a file, a "blank"
                    track appears letting you know that IGB recognized your file.
                </li>
            </ul>
            <hr>

            <h3>
                Feb 6, 2012: New tutorials on working with RNA-Seq data
                available
            </h3>
            <br>

            <p>
                Many users have asked us how to create files for visualization in
                IGB. This can be a daunting task, particularly when working with
                results from high-throughput sequenicng experiments.
            </p>

            <p>
                To help users and the bioinformatics personnel who support their
                research, we have published two new tutorials describing how to use
                bioinformatics tools such as 
                <a href=
                   "http://tophat.cbcb.umd.edu/">
                    tophat</a>, 
                <a href=
                   "http://samtools.sourceforge.net/">
                    samtools</a>, Jim Kent's 
                <a href=
                   "http://hgdownload.cse.ucsc.edu/admin/exe/">
                    faToTwoBit</a>, and
                <a href="http://samtools.sourceforge.net/tabix.shtml">
                    tabix</a> to process and share next-generation sequencing data sets with lab members
                and collaborators using IGB and the simple IGB QuickLoad system.
            </p>

            <p>
                To view the tutorials, visit the IGB Developer's Guide section
                titled 
                <b>
                    <a href=
                       "https://wiki.transvar.org/confluence/x/0gCjAQ">
                        Sharing data
                        sets
                    </a>
                </b>.
            </p>

            <p>
                In the next few weeks we hope to add more tutorials that will help
                users make better use of their data and also get more familiar with the
                rich set of features IGB offers for visualization and analysis of
                genomic data sets.
            </p>

            <p>
                If you have questions or suggestions, please let us know. Email
                aloraine - at - uncc.edu with your comments and requests. We will do
                our best to respond quickly and thoroughly to all enquiries.
            </p>
            <hr>

            <h3>
                Feb 3, 2012: NetAffx data back on line, hosted at UNC
                Charlotte
            </h3>
            <br>

            <p>
                The data formerly available via the NetAffx DAS2 data source,
                including probe set alignments for Affymetrix array products, RefSeq
                genes, and chromosome bandingb patterns ("cytobands"), are now
                available again from 
                <a href="http://igbquickload.org/das2">
                    the new
                    DAS2 data source
                </a>
                hosted at 
                <a href=
                   "http://www.igbquickload.org">
                    IGBQuickLoad.org</a>.
            </p>

            <p>
                As a convenience to users, we have re-activated automatic loading of
                gene models (the RefSeq genes data sets) and cytological bands for
                certain genomes, including the human genome. If you experience any
                further problems with these data sets, please let us know - email
                aloraine --at-- uncc.edu.
            </p>

            <p>
                Many thanks are due to Scott Wood (UNCC 
                <a href=
                   "http://cci.uncc.edu/">
                    College of Computing and Informatics</a>) who
                helped us a huge amount in setting up the new site. Thanks are also due
                to IGB developer Hiral Vora who updated the DAS2 servlet code to
                support the Affymetrix data. Last but not least, we are also grateful
                to Dr. Ron Shigeta of Affymetrix for sending us the Affymetrix data.
                Thank you Ron, Scott, and Hiral for your efforts on behalf of the IGB
                user community!
            </p>
            <hr>

            <h3>
                January 31, 2012: Temporary service outage for
                IGBQuickload.org
            </h3>
            <br>

            <p>
                We apologize for the inconvenience.
            </p>

            <p>
                Starting around 1 pm EST today, the IGB QuickLoad resource will
                become temporary unavailable.
            </p>

            <p>
                We are adding more memory to the server and installing some new
                software. We don't expect to be off-line for more than a couple of
                hours at the most.
            </p>

            <p>
                You may have noticed that the NetAffx DAS2 data source is no longer
                responding. Unfortunately, there was an irrecoverable failure in the
                DAS2 data source hosted at Affymetrix. We are working on a solution and
                should have the data sets Affymetrix was hosting back on line; we are
                moving the data to a virtual server residing at UNC Charlotte.
            </p>

            <p>
                In the meantime, we've deactivated the NetAffx DAS2 data source as
                one of the default IGB data sources. Hopefully by the end of the week
                you will have full access to the data once again.
            </p>
            <hr>

            <h3>
                January, 2012: IGB 6.7 alpha available on BioViz Web site.
            </h3>
            <br>

            <p>
                A preview (alpha) release version of IGB 6.7 is now available on the
                BioViz Web site. Just visit the 
                <a href="download.html">
                    Download
                    Page</a> to get a copy.
            </p>

            <p>
                <b>
                    Note to programmers
                </b>
                : The IGB 6.7 release version is now
                branched in the subversion repository on sourceforge.
            </p>

            <p>
                New features coming soon in IGB 6.7 include:
            </p>

            <ul>
                <li>
                    <b>
                        Better data loading
                    </b>
                    : When you access a file, a "blank"
                    track appears letting you know that IGB recognized your file.
                </li>

                <li>
                    <b>
                        Better bookmarks
                    </b>
                    : Now, when you make a bookmark, you can
                    add a note describing it. You can also more easily edit, move,
                    delete, and share your bookmarks.
                </li>

                <li>
                    <b>
                        All-new start screen
                    </b>
                    : We added shortcuts to the IGB
                    start screen, making it easier than ever to find your genome of
                    interest.
                </li>

                <li>
                    <b>
                        Better support for image export
                    </b>
                    : We improved the way
                    that images are created, adding the ability to increase or decrease
                    image quality. Now it's easy to create high-quality (300 dpi or
                    better) images for publication.
                </li>
            </ul>
            <hr>

            <h3>
                October, 2011: IGB 6.6 released
            </h3>
            <br>

            <p>
                IGB version 6.6 is released on the BioViz Web site. To read about
                new features and improvements in this new version of IGB, see 
                <a href=
                   "http://wiki.transvar.org/display/igbman/New+in+IGB+6.6">
                    New
                    in IGB 6.6</a> in the 
                <a href=
                   "http://wiki.transvar.org/display/igbman/Home">
                    IGB User's
                    Guide</a>.
            </p>

            <p>
                New features in IGB 6.6 include:
            </p>

            <ul>
                <li>
                    <b>
                        Improved Data Access Panel
                    </b>
                    . You can now use the Data
                    Access Panel to change track colors, hide or show tracks, or
                    double-click in the Track Name column to re-name a track.
                </li>

                <li>
                    <b>
                        New Options for Configuring Tracks
                    </b>
                    . You can now use
                    arrows or color to indicate strand for features and/or sequence
                    read alignments. You can also increase or decrease the font track
                    label font. Choose File-&gt;Preferences to configure tracks, or
                    just right-click a track label and choose the Configure
                    option.
                </li>
            </ul>
            <hr>

            <h3>
                June, 2011: IGB team at ICAR 2011
            </h3>
            <br>

            <p>
                The IGB team is hosting a booth at the 
                <a href=
                   "http://www.union.wisc.edu/arabidopsis/">
                    International Conference on
                    Arabidopsis Research
                </a>
                in Madison, WI. We'll be giving one-on-one
                demos and answering questions about using IGB in research.
            </p>

            <p>
                Ann gave a talk on IGB - slides are 
                <a href=
                   "./loraine-icar-2011.pptx">
                    available here</a>. David Norris and the IGB
                team presented 
                <a href="./icar-2011-poster.pdf">
                    a poster</a>.
            </p>
            <hr>

            <h3>
                April, 2011: IGB 6.5 released
            </h3>
            <br>

            <p>
                IGB version 6.5 is now available for download from the BioViz Web
                site. For details, see: 
                <a href=
                   "https://wiki.transvar.org/confluence/display/igbman/New+in+IGB+6.5">
                    New
                    in IGB 6.5
                </a>
                in the IGBbUser's Guide.
            </p>

            <p>
                New features available in IGB 6.5 include:
            </p>

            <ul>
                <li>
                    <b>
                        Sequence Viewer
                    </b>
                    . View, translate and copy sequence in a
                    new user-friendly interface.
                </li>

                <li>
                    <b>
                        More ways to zoom and pan.
                    </b>
                    Use the new grab tool to move
                    the display up or down, left or right.
                </li>

                <li>
                    <b>
                        Insertions and deletions
                    </b>
                    . View insertions and deletions
                    in alignment (BAM) files.
                </li>

                <li>
                    <b>
                        Expanded support for plant genomes
                    </b>
                    . The IGB QuickLoad
                    site now hosts data for publicly available plant geomes including
                    grape, soy, rice, Arabidopsis lyrata, and more.
                </li>
            </ul>
            <hr>

            <h3>
                January 15-19, 2011: IGB at the Plant Animal Genomes
                Conference
            </h3>
            <br>

            <p>
                Ann is attending 
                <a href="http://www.intl-pag.org/">
                    PAG XIX
                </a>
                and
                giving a live demo of IGB at 10:30 am on Wednesday in the Computer Demo
                Room. Please read 
                <a href=
                   "http://www.intl-pag.org/19/abstracts/C02_PAGXIX_909.html">
                    the
                    abstract
                </a>
                from the demo. We hope to see you there!
            </p>
            <hr>

            <h3>
                November 22, 2010: IGB 6.4 released
            </h3>
            <br>

            <p>
                Thank you to everyone who tried IGB 6.4 Alpha and gave us your
                feedback. IGB 6.4 is now available and incorporates many of your
                suggestions.
            </p>
            <hr>

            <h3>
                October 22, 2010: IGB 6.4 Alpha released - Please try it
                out!
            </h3>
            <br>

            <p>
                IGB 6.4 ALPHA is now available for preview on BioViz. Please visit
                the 
                <a href="IGB_6_4.shtml">
                    IGB 6.4 preview page
                </a>
                to download a
                copy.
            </p>

            <p>
                Thank you very much to everyone who responded to the IGB survey back
                in April 2010. You contributed many great ideas to improve IGB and we
                were able to implement many of them in IGB 6.4. Check it out and let us
                know what you think!
            </p>

            <p>
                As always, please report any problems you encounter - just choose
                "Report a bug" or "Request a feature" from the IGB Help Menu. We now
                have a dedicated testing group for IGB, but we continue to rely on YOU
                for help making certain IGB performs well for your research.
            </p>

            <p>
                New features available in IGB 6.4 include:
            </p>

            <ul>
                <li>
                    <b>
                        Better Bookmarks
                    </b>
                    . You can now bookmark data sets and
                    current view settings. With new and improved IGB bookmarks, you can
                    retrieve not just the region, but also the data together with
                    color, display name, and other settings.
                </li>

                <li>
                    <b>
                        Data Access tab
                    </b>
                    You can now delete tracks. You can also
                    refresh one track at a time without having to change the Load Mode
                    settings in the Data Access Panel.
                </li>

                <li>
                    <b>
                        Tracks and Tiers
                    </b>
                    . When you mouse over a track label or
                    feature in a track, you'll see a tooltip showing the same
                    information as what appears in the Selection Info table. But if
                    it's too much information, you can turn off the tooltips using the
                    View menu.
                </li>

                <li>
                    <b>
                        Searching
                    </b>
                    . Now searching works for all feature
                    properties, not just names. Note: This only works for data that
                    you've already loaded.
                </li>

                <li>
                    <b>
                        Loading data - 
                        <i>
                            for power users
                        </i>
                    </b>
                    . If you're sharing
                    a genome using an IGB QuickLoad data server, you can now use BNIB
                    or 2BIT sequence file formats on your site and your users will be
                    able to click the Load Sequence in View button to get just the
                    sequence in view. (This happens via partial HTTP requests.) We also
                    now support PSLX file format, an alignment format similar to PSL
                    but which also includes query sequence.
                </li>

                <li>
                    <b>
                        Scripting - 
                        <i>
                            for power users
                        </i>
                    </b>
                    . We fixed some
                    synchronization issues that were forcing some of you to tell IGB to
                    wait before completing a script.
                </li>
            </ul>
            <hr>

            <h3>
                September 27, 2010: IGB 6.3.1 released
            </h3>
            <br>

            <p>
                IGB 6.3.1 incorporates several bugfixes to IGB 6.3.
            </p>
            <hr>

            <h3>
                September 22, 2010: Alyssa Gulledge joins IGB team
            </h3>
            <br>

            <p>
                The IGB team here at UNC Charlotte welcomes Alyssa Gulledge, Ph.D.,
                our new testing, outreach, and documentation coordinator. Before
                joining us, Alyssa worked as a scientist and postdoctoral researcher at
                UNC Chapel Hill and UNC Charlotte.
            </p>
            <hr>

            <h3>
                July 21, 2010: IGB 6.3 released
            </h3>
            <br>

            <p>
                Some new client features are listed below:
            </p>

            <ul>
                <li>
                    BAM parser

                    <ul>
                        <li>
                            (Be sure you've sorted and indexed your BAM
                            files!)
                        </li>

                        <li>
                            Shows gapped alignments and properties.
                        </li>

                        <li>
                            When zoomed in, shows residues. If partial or full
                            reference sequence is loaded, shows only differing
                            residues.
                        </li>
                    </ul>
                </li>

                <li>
                    Local files and QuickLoad: allow partial (chromosome) feature
                    loading for most formats.
                </li>

                <li>
                    Local files and QuickLoad: allow region-based feature loading
                    for BAM, BED, Wiggle, Sgr, Gr, Fasta, and others.
                </li>

                <li>
                    Drag and Drop for files and (file or QuickLoad) URLs.
                </li>

                <li>
                    "Load from URL" menu item. Allows region-based feature loading
                    for BAM.
                </li>

                <li>
                    Scripting options for IGB. See this 
                    <a href=
                       "http://wiki.transvar.org/display/igbman/Scripting">
                        Wiki
                        page
                    </a>
                    for more details.
                </li>

                <li>
                    Deletion of tracks.
                </li>

                <li>
                    Ensembl external browser (also loadable via DAS).
                </li>

                <li>
                    Load View: combo box hints for species (common name) and
                    genomes (synonyms).
                </li>

                <li>
                    Whole genome view for all organisms.
                </li>

                <li>
                    Sort chromosome list by contig/chromosome size
                </li>

                <li>
                    Centralized server caching of common requests from slower
                    servers (mostly UCSC and Ensembl).
                </li>
            </ul>

            <p>
                As always, old versions of IGB are also available from our 
                <a href=
                   "download.html">
                    download
                </a>
                page.
            </p>
            <hr>

            <h3>
                July 19, 2010: Slime mold Dictyostelium discoideum strain Ax4
                genome data added
            </h3>
            <br>

            <p>
                Using data and advice kindly provided by the 
                <a href=
                   "http://www.dictybase.org/About_dictyBase/">
                    dictyBase team</a>, we
                created BED and sequence data files so that Dictyostelium researchers
                can view their data in IGB. The BioViz DAS2 data source now provides
                DAS2-style access to Dicty Ax4 sequence data, and the BioViz QuickLoad
                site now provides access to BED format files for dictyBase
                Dictyostelium discoideum strain Ax4 gene annotations. When you view the
                data in IGB, you can right-click on gene models and choose the
                "DictyBase" option to view the corresponding record in dictyBase.
            </p>

            <p>
                For more information about the data, visit the 
                <a href=
                   "http://www.transvar.org/~aloraine/genomes.pub/quickload/D_discoideum_Ax4_May_2009/">
                    Dicty May 2009 QuickLoad Web directory</a>.
            </p>
            <hr>

            <h3>
                June 25, 2010: IGB 6.3 Beta released
            </h3>
            <br>
            <hr>

            <h3>
                April 8, 2010: IGB 6.2.1 released
            </h3>
            <br>

            <p>
                Some new client features are listed below:
            </p>

            <ul>
                <li>
                    DAS/1 support
                </li>

                <li>
                    External UCSC browser tab
                </li>

                <li>
                    Localization
                </li>

                <li>
                    GUI and Preferences bugfixes and enhancements
                </li>

                <li>
                    USeq support
                </li>
            </ul>
            <hr>

            <h3>
                March 26, 2010: IGB at the ASPB meeting
            </h3>
            <br>

            <p>
                Ann gave an IGB demo at the Mid-Atlantic Section American Society of
                Plant Biologists 
                <a href=
                   "http://www.life.umd.edu/labs/ATRIUM/Symposium/Spring10mtg.html">
                    Annual
                    Spring Meeting</a>, held at 
                <a href="http://www.bowiestate.edu/">
                    Bowie
                    State University
                </a>
                in Maryland. Slides from the talk are 
                <a href=
                   "Loraine-ASPB-Mar-2010.pdf">
                    available on-line</a>.
            </p>
            <hr>

            <h3>
                January 9-13, 2010: IGB at the Plant Animal Genomes
                Conference
            </h3>
            <br>

            <p>
                Ann attended 
                <a href="http://www.intl-pag.org/">
                    PAG XVIII</a>, gave
                a live demo of IGB, and presented 
                <a href=
                   "http://www.transvar.org/posters/PAG2010_igb_poster.pdf">
                    this
                    poster</a>.
            </p>
            <hr>

            <h3>
                January 6, 2010: IGB 6.1 released
            </h3>
            <br>

            <p>
                Some new client features are listed below:
            </p>

            <ul>
                <li>
                    major memory improvements for graphs (SGR, Wiggle, etc.)
                </li>

                <li>
                    ability to parse large Wiggle files
                </li>

                <li>
                    better handling for large files over network
                </li>

                <li>
                    customizable coloring for residues
                </li>

                <li>
                    "whole chromosome" mode for DAS/2; only activated when data is
                    refreshed
                </li>

                <li>
                    enhanced DAS/1 support (Ensembl can now be accessed)
                </li>

                <li>
                    assorted GUI enhancements and bugfixes
                </li>
            </ul>
            <hr>

            <h3>
                September 29, 2009: IGB 6.0 released
            </h3>
            <br>

            <p>
                Some new features are listed below:
            </p>

            <p>
                IGB client features:
            </p>

            <ul>
                <li>
                    compiled with Java 6
                </li>

                <li>
                    enhanced data source preferences page
                </li>

                <li>
                    unification of "Pattern Search", "Name Search", and "Annotation
                    Browser" tabs
                </li>

                <li>
                    auto-loading of features (if specified by DAS/2 server)
                </li>

                <li>
                    data provider hyperlinks and icons
                </li>

                <li>
                    feature hyperlinks and description tooltips
                </li>

                <li>
                    QuickLoad annots.xml file (similar to DAS/2)
                </li>

                <li>
                    GFF and BED parsing bugfixes
                </li>

                <li>
                    warnings for chromosome conflicts
                </li>

                <li>
                    assorted GUI enhancements and bugfixes
                </li>
            </ul>

            <p>
                DAS/2 server features:
            </p>

            <ul>
                <li>
                    file indexing: reduces memory requirements by up to 90%, often
                    reducing startup time as well.
                </li>

                <li>
                    improved ID searches: reduces network traffic, thus speeding up
                    client-side search.
                </li>

                <li>
                    warnings for chromosome conflicts
                </li>

                <li>
                    additional annots.xml properties
                </li>
            </ul>
            <hr>

            <h3>
                September 11, 2009: IGB 
                <a href=
                   "http://en.wikipedia.org/wiki/Integrated_Genome_Browser">
                    Wikipedia
                    article
                </a>
                published
            </h3>
            <br>
            <hr>

            <h3>
                August 25, 2009: Genoviz SDK article published in 
                <i>
                    BMC
                    Bioinformatics
                </i>
            </h3>
            <br>

            <table style="border-collapse: collapse; line-height: 1.1em;">
                <tbody>
                    <tr style="vertical-align: top;">
                        <td style="padding-left: 4pt;">
                            Helt, G.A., Nicol, J.W.,
                            Erwin, E., Blossom, E., Blanchard, S.G., Chervitz, S.A.,
                            Harmon, C. &amp; Loraine, A.E. The Genoviz Software
                            Development Kit: A Java toolkit for building genomics
                            visualization applications. 
                            <span style=
                                  "font-style: italic;">
                                BMC Bioinformatics
                            </span>
                            <span style=
                                  "font-weight: bold;">
                                10</span>, 266 (2009).doi:
                            <a href=
                               "http://dx.doi.org/10.1186/1471-2105-10-266">
                                10.1186/1471-2105-10-266
                            </a>
                        </td>
                    </tr>
                </tbody>
            </table>
            <hr>

            <h3>
                August 4, 2009: IGB article published in
                <i>
                    Bioinformatics
                </i>
            </h3>
            <br>

            <table style="border-collapse: collapse; line-height: 1.1em;">
                <tbody>
                    <tr style="vertical-align: top;">
                        <td style="padding-left: 4pt;">
                            Nicol, J.W., Helt, G.A.,
                            Blanchard, S.G., Raja, A. &amp; Loraine, A.E. The
                            Integrated Genome Browser: Free software for distribution
                            and exploration of genome-scale data sets. 
                            <span style=
                                  "font-style: italic;">
                                Bioinformatics
                            </span>
                            (2009).doi:
                            <a href=
                               "http://dx.doi.org/10.1093/bioinformatics/btp472">
                                10.1093/bioinformatics/btp472
                            </a>
                        </td>
                    </tr>
                </tbody>
            </table>
            <hr>

            <h3>
                June 16, 2009: IGB version 5.5 released.
            </h3>
            <br>

            <p>
                This release features improved memory management, allowing users to
                load even more data into the viewer, as well as further streamlining of
                the user interface. Other improvements include support for adding new
                Quickload or DAS servers via the Preferences Tab and better image
                export support (print to file) for many formats, including PDF, PNG,
                EPS, PS, and others. For more details, see the 
                <a href=
                   "http://genoviz.sourceforge.net/release_notes/igb_release.html">
                    IGB
                    Release Notes</a>.
            </p>
            <hr>

            <h3>
                May, 2009: The IGB team welcomes four new students and bids
                farewall to Lokeshvar
            </h3>
            <br>

            <p>
                The IGB team welcomes two new students from the UNC Charlotte
                Professional Science Masters Program who are doing their internships
                with us this summer. Archana Raja is working on documentation, testing
                the user interface, and collecting tiling array data sets for our DAS2
                server. Nathaniel Watson is collecting data sets from sequenced plant
                genomes so that we can support other plant species in addition to
                <i>
                    Arabidopsis
                </i>
                . We also welcome two new undergraduate summer
                interns: Kristen Sagliani, a chemistry/math double major from UNC
                Charlotte and Roshonda Barner, a math/economics double major from NC A
                &amp; T. Kristen and Roshonda are helping develop an on-line
                programming course that demonstrates bioinformatics programming
                concepts using data set from the Bioviz DAS2 and Quickload servers. We
                also bid farewell to Lokeshvar, who starts a new internship at Wachovia
                this summer. Good luck with your new job and thank you for your great
                work on IGB.
            </p>
            <hr>

            <h3>
                March 26, 2009: IGB version 5.4 released
            </h3>
            <br>

            <p>
                We are pleased to announce the release of version 5.4 of IGB, which
                features a much improved mechanism for loading data into the browser.
                Users can now access Affymetrix DAS2, Bioviz DAS2, and Bioviz Quickload
                data sources using the same easy-to-use interface. For details on how
                this works, see the FAQ item describing how to load data into IGB.
            </p>
            <hr>

            <h3>
                March 12, 2009: Welcome aboard, Lokeshvar
            </h3>
            <br>

            <p>
                The IGB team welcomes Lokeshvar Iyanar, who starts his new position
                as test and automation engineer today. Lokeshvar is working toward his
                masters' degree in Computer Science from the University of North
                Carolina at Charlotte. Previously, he worked as a Performance Test
                Analyst at for Wachovia and as a Student Technical Specialist at the
                Mosaic Lab at UNCC.
            </p>
            <hr>

            <h3>
                March 9, 2009: TableView Support
            </h3>
            <br>

            <p>
                We have added a new Java Launch page for James (Jim) Johnson's
                <a href=
                   "http://bioinformatics.oxfordjournals.org/cgi/content/abstract/19/10/1292">
                    TableView
                </a>
                application, originally hosted at the University of
                Minnesota's Center for Computational Genomics and Bioinformatics, which
                closed in 2008. Jim's software is very useful for visualizing
                interactive scatter plots, a feature many IGB users may find helpful
                when working with expression array data. To download TableView, visit
                <a href="http://www.transvar.org/TableView.html">
                    TableView launch
                    page</a>. Let us know if you have questions or comments!
            </p>
            <hr>

            <h3>
                Jan. 7, 2009: IGB supports Arabidopsis EST data.
            </h3>
            <br>

            <p>
                We used Jim Kent's genomic alignment tool 
                <a href=
                   "http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11932250">
                    blat
                </a>
                to align around 1.5 million Arabidopsis thaliana ESTs
                harvested from Genbank in December 2008 onto the TAIR8 Arabidopsis
                genome assembly. We then applied a simple quality-control filter,
                following the same specification the UCSC Genome Informatics team uses
                to generate their "all EST" data set, with one important difference: We
                divided the alignments into two separate data sets:
            </p>

            <ul>
                <li>
                    <b>
                        single mappers:
                    </b>
                    ESTs that aligned once to the
                    genome
                </li>

                <li>
                    <b>
                        multi-mappers:
                    </b>
                    ESTs that aligned more than once to the
                    genome
                </li>
            </ul>
            <hr>

            <h3>
                Dec. 10, 2008: IGB poster at UNC Charlotte Bioinformatics and
                Genomics Retreat
            </h3>
            <br>

            <p>
                IGB developer John Nicol presented a poster describing his
                contributions to the IGB and Genoviz projects at the UNC Charlotte
                Bioinformatics and Genomics Retreat. Here John discusses his work with
                an attendee at the event.
            </p>

            <p>
                <img alt="IGB Poster" src="img/jn_big_retreat2.jpg" style=
                     "max-width:60%;">
            </p>

            <p>
                John Nicol and BiG Retreat attendee
            </p>
            <br>
            <hr>

            <div id="footer" class="container">
            </div>
        </div>
        <!-- /container -->

        <script src="//code.jquery.com/jquery-1.11.2.min.js"></script>

        <!-- scripts concatenated and minified via ant build script-->
        <script src="js/libs/bootstrap/bootstrap.min.js"></script>
        <script src="js/script.js?v=12392829">
        </script>
        <script src="bower_components/marked/marked.min.js"></script>
         <script src="bower_components/lightbox2/dist/js/lightbox.min.js"></script>
        <script type="text/javascript">
            $(window).load(function () {
                var newsMarkdown = "news.md ";
                $.get(newsMarkdown, function (data) {
                    $("#markupOutput").html(marked(data));
                });
            });

        </script>
        <!-- end scripts-->

        <script>
            (function (i, s, o, g, r, a, m) {
                i['GoogleAnalyticsObject'] = r;
                i[r] = i[r] || function () {
                    (i[r].q = i[r].q || []).push(arguments)
                }, i[r].l = 1 * new Date();
                a = s.createElement(o),
                        m = s.getElementsByTagName(o)[0];
                a.async = 1;
                a.src = g;
                m.parentNode.insertBefore(a, m)
            })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');

            ga('create', 'UA-33977587-2', 'bioviz.org');
            ga('send', 'pageview');
        </script>

    </body>
</html>
